### DISCUSSION TITLE
Link Shortcut

### DISCUSSION THREAD

**Discussion by baswein** [outline/outline#2385] [Self Hosting]
Hi, 
Excited to discover Outline! 
I am trying to figure out how to link in the document. 
In the shortcut list it says it is ctrl+K but that brings up the search function. Is that right? I'm on firefox with windows 10. 
I have tried to google this but unfortunately outline link is too generic as a phrase. Is there a google phrase that I could use to specify that I am looking for the outline documentation editor?  
Thanks

**Comment by tommoor** [comment]
`ctrl+k` is correct but you must have focus in the editor and it will allow creation of a link, otherwise it _does_ go to search (to match lots of other apps).

**Comment by baswein** [comment]
Thanks @tommoor  - 
Here is a gif of what is happening for me. Also previously the Formatting Toolbar wasn't showing up so I didn't see that there was another way to link. Now I can see how the linking works with the Formatting Toolbar and don't really need the shortcut.  
![outline_link](https://user-images.githubusercontent.com/15817745/127346782-024568d2-da6a-4416-b0da-7405072b7518.gif)
I also tried this in chrome and got the same result. 
Thanks Again!

**Comment by tommoor** [comment]
We have seen a number of issues with event handling on Firefox acting
differently depending on the platform- it could be Windows+ff specific

**Reply by baswein** [reply]
I did try it in chrome and it did the same thing. Just thought you should know. 
I have been really enjoying Outline!
